draft-wang-ppm-differential-privacy.md

```diff
@@ -260,7 +260,7 @@ common use cases, lifts DAP to a protocol that also achieves DP.
 implementations of mechanisms and policies can be found at
 https://github.com/wangshan/draft-wang-ppm-differential-privacy.]
 
-Multi-party computation systems like the Distributed Aggregration Protocol
+Multi-party computation systems like the Distributed Aggregation Protocol
 {{!DAP=I-D.draft-ietf-ppm-dap-05}} enable secure aggregation of measurements
```

Comment:
Just noticed: we need to update DAP and VDAF references.

Reply:
Good call, will do